.github/workflows/bounty-payout.yml

```
@@ -0,0 +1,109 @@
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for linked issues with bounties
+        id: check-issues
+        uses: actions/github-script@v6.1.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const pr = context.payload.pull_request;
+            const prBody = pr.body || '';
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)